### ISSUE TITLE
Docker + GitHub Actions

### ISSUE THREAD

**Issue by MuhammadKhalilzadeh** [bluewave-labs/verifywise#315]
Hello @HarshP4585 

In this task, as we discuss, we will use the ability of GitHub actions in order to have the Docker image creation (update) automated.

- [x] #547
- [x] And an image for the database

Regards, Mo.

**Comment by HarshP4585** [comment]
Added Docker for frontend, backend and the database with the `docker-compose` file

@MuhammadKhalilzadeh @gorkem-bwl Do I need to integrate GitHub Actions as part of this release?

**Comment by gorkem-bwl** [comment]
Yes, we can include GitHub actions. If it gets done for this release, that would be great. If not, we are also fine. Adding 1.0 milestone for now and let's see how it goes.
